applications/matter_bridge/README.rst

```diff
@@ -373,6 +376,8 @@ After building the sample and programming it to your development kit, complete t
                I: Adding OnOff Light bridged device
                I: Added device to dynamic endpoint 3 (index=0)
 
+            Note that in the case of Environmental Sensor, two endpoints are created: the first one implements the Temperature Sensor, and the second one implements Humidity Sensor.
```

Comment:
```suggestion
            For the Environmental Sensor, two endpoints are created: one implements the Temperature Sensor, and the other implements the Humidity Sensor.
```
I think this reads a bit better. Unless the order of the endpoints is very important, in which case it should "the first one" and "the second" instead of "one" and "the other", respectively.

Reply:
I agree, the order here is not important.